Update contributor-orientation.php

sereedmedia opened the pull request:
This update adds all teams to each section so all teams intended to display are possible options at the end. Also adds the Hosting Team, edits the Photos Team URL to their main Handbook page, and edits the title of each results to say "Get started with [team name]".

sereedmedia committed:
Update contributor-orientation.php

This update adds all teams to each section so all teams intended to display are possible options at the end. Also adds the Hosting Team, and edits the title of each results to say "Get started with [team name]".

sereedmedia commented:
This should close Issues  #3  , #8 , and #9

ryelle commented:
Everything still works, the new pathways are good. Thanks for sorting the team lists 🙂 

I still don't see Meta, Plugins, or Tide as possibilities, despite being in the full teams list. Should we remove them from `get_teams` entirely, if they're intentionally not included?

I also don't see Openverse in the full teams list (in `get_teams()`), so I think that still needs to be added from the info in #9.

It looks a little repetitive now with "Get started with…" for each team, but that's just an opinion.

<img width="400" src="https://github.com/WordPress/wporg-contributor-orientation/assets/541093/65c11565-11c6-4fab-bf91-d230fdd450e8">

ryelle commented:
One last nitpicky comment - should this use [the WP Brand Writing Style](https://make.wordpress.org/marketing/handbook/resources/style-guide-and-brand-book/#formatting)? If so, the "Get Started…" headings should be sentence case, "Get started with…"

sereedmedia commented:
I think you are correct re: the Style Guide headings sentence case. I will update.

I personally think that the "Get Started" heading is doing double-duty, serving both as a Heading and as a Call to Action, because just having the title of the team is just a list and not particularly welcoming or action-inspiring. Additionally, the heading will just be on the suggestions, which isn't intended to display a lot of options (though I don't know what is being output currently).

Yes, the exclusion of Meta, Tide and Plugins is intentional. I didn't take them out altogether because I personally think they should be added back in at some point. Also, leaving them in made it evident they weren't there. Perhaps I should add some comments explaining that choice?

I will check re: Openverse, really thought it was in there already.

And YAY for alpha order! LOL.

sereedmedia committed:
Update contributor-orientation.php for 2nd round of fixes for MVP tool

- Added Openverse as team option
- Edited case of "Get started ..." links
- Added a comment explaining that the exclusion of Meta, Tide, and Plugins is intentional

sereedmedia commented:
Updated PR to incorporate @ryelle's feedback

ryelle committed:
Fix syntax issues

ryelle committed:
Update Openverse icon

ryelle commented on contributor-orientation.php:
```diff
@@ -396,124 +487,138 @@ function get_pages() {
+			'name'        => esc_html__( 'Get started with the Openverse Team', 'wporg' ),
+			'description' => esc_html__( 'The Openverse Team implements new features and new media types; maintains the public API and front-end search engine; and develops WordPress integrations to share Openverse with the entire WordPress community.', 'wporg' ),
+			'icon'        => '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><rect x="0" fill="none" width="20" height="20"/><g><path d="M14.9 9c0-.3.1-.6.1-1 0-2.2-1.8-4-4-4-1.6 0-2.9.9-3.6 2.2-.2-.1-.6-.2-.9-.2C5.1 6 4 7.1 4 8.5c0 .2 0 .4.1.5-1.8.3-3.1 1.7-3.1 3.5C1 14.4 2.6 16 4.5 16h10c1.9 0 3.5-1.6 3.5-3.5 0-1.8-1.3-3.3-3.1-3.5z"/></g></svg>',
+			'url'         => 'https://make.wordpress.org/openverse/handbook/new-contributor-guide/'
```
This is a code style issue, but items in an array should end with commas
```suggestion
			'url'         => 'https://make.wordpress.org/openverse/handbook/new-contributor-guide/',
```

sereedmedia replied:
yes

ryelle commented on contributor-orientation.php:
```diff
@@ -396,124 +487,138 @@ function get_pages() {
+			'description'   => esc_html__( 'Tide Team manages the Tide project, a series of automated tests run against every plugin and theme in the directory and displays PHP compatibility and test errors/warnings in the directory.', 'wporg' ),
+			'icon'          => '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><rect x="0" fill="none" width="20" height="20"/><g><path d="M17 7.2V3H3v7.1c2.6-.5 4.5-1.5 6.4-2.6.2-.2.4-.3.6-.5v3c-1.9 1.1-4 2.2-7 2.8V17h14V9.9c-2.6.5-4.4 1.5-6.2 2.6-.3.1-.5.3-.8.4V10c2-1.1 4-2.2 7-2.8z"/></g></svg>',
+			'url'           => 'https://make.wordpress.org/tide/feedback-support/',
+		),
```
This was missing the final parenthesis, which caused a parse error on the site.

```suggestion
		),
	);
```

sereedmedia replied:
GAH!

ryelle commented on contributor-orientation.php:
```diff
@@ -396,124 +487,138 @@ function get_pages() {
+		'openverse' => array(
+			'name'        => esc_html__( 'Get started with the Openverse Team', 'wporg' ),
+			'description' => esc_html__( 'The Openverse Team implements new features and new media types; maintains the public API and front-end search engine; and develops WordPress integrations to share Openverse with the entire WordPress community.', 'wporg' ),
+			'icon'        => '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><rect x="0" fill="none" width="20" height="20"/><g><path d="M14.9 9c0-.3.1-.6.1-1 0-2.2-1.8-4-4-4-1.6 0-2.9.9-3.6 2.2-.2-.1-.6-.2-.9-.2C5.1 6 4 7.1 4 8.5c0 .2 0 .4.1.5-1.8.3-3.1 1.7-3.1 3.5C1 14.4 2.6 16 4.5 16h10c1.9 0 3.5-1.6 3.5-3.5 0-1.8-1.3-3.3-3.1-3.5z"/></g></svg>',
```
I also updated the openverse logo, since this is the hosting cloud icon.

sereedmedia replied:
omg, thank you! i tried to get that SVG right tooooo many times lol.

ryelle commented on contributor-orientation.php:
```diff
@@ -396,124 +487,138 @@ function get_pages() {
+			'icon'        => '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><rect x="0" fill="none" width="20" height="20"/><g><path d="M14.9 9c0-.3.1-.6.1-1 0-2.2-1.8-4-4-4-1.6 0-2.9.9-3.6 2.2-.2-.1-.6-.2-.9-.2C5.1 6 4 7.1 4 8.5c0 .2 0 .4.1.5-1.8.3-3.1 1.7-3.1 3.5C1 14.4 2.6 16 4.5 16h10c1.9 0 3.5-1.6 3.5-3.5 0-1.8-1.3-3.3-3.1-3.5z"/></g></svg>',
+			'url'         => 'https://make.wordpress.org/openverse/handbook/new-contributor-guide/'
+		),
+/** 
+* Note: The Plugins, Meta and Tide teams are intentionally not currently displaying as suggested teams because they do not have sufficient new contributor onboarding processes 
+*/			
+		'plugins' => array(
```
I don't think this one is technically an issue, but it's easier to read the comment if it's at the same indentation as the rest of the array.

There is also a code style issue here because the comment has some trailing space. If you've got this project set up locally, you can run `npm run lint:php` to check that. A github action could also be added to check this on each PR.

```suggestion

		/**
		 * Note: The Plugins, Meta, and Tide teams are intentionally not currently
		 * displaying as suggested teams because they do not have sufficient new
		 * contributor onboarding processes.
		 */
		'plugins' => array(
```

sereedmedia replied:
Ah! thank you for updating and the tip about the trailing space!
